### DISCUSSION TITLE
Cannot instal Homebrew on Macbook air

### DISCUSSION THREAD

**Discussion by Sleepycatfuji** [Homebrew/discussions#2683] [Getting started]
Hi, I figured it out! 
I just typed my computer password and it works now... sorry for bothering everyone

I tried installing Homebrew through the terminal but failed because it keeps asking for a password. It has a little key photo and I can't seem to type in anything.
Below is a rundown of what appears on my console.

https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Password:
Sorry, try again.
Password:
sudo: 1 incorrect password attempt
Need sudo access on macOS (e.g. the user XXXX needs to be an Administrator)!
xxxxxxxx-MacBook-Air ~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Password:
Sorry, try again.
Password:
Sorry, try again.
Password:

**Comment by ghost** [comment]
~ ❯ brew install node                                                  20:51:42
fatal: Could not resolve HEAD to a revision
Warning: No available formula with the name "node".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.


I have error

**Reply by SMillerDev** [reply]
Completely unrelated to the issue that was posted, but try `brew update-reset`
